zinc/src/sbt-test/source-dependencies/subproject-pipelining-mixed/use/D.scala

```diff
@@ -0,0 +1,7 @@
+package example
+
+class D extends C {
+  override def hi(): Int = {
+    super.hi() + 1
+  }
+}
```

Comment:
How does this reproduce 918?  Without the last commit this test seems to pass.

Reply:
This PR was stacked on top of https://github.com/sbt/zinc/pull/920 since that what I was using to test locally. This PR is aimed to fix https://github.com/sbt/zinc/issues/911